src/pages/PrintOnlyScreen.tsx

```diff
@@ -143,7 +142,7 @@ const PrintOnlyScreen = ({
             />
           </p>
           <h1>
-            <Loading>Printing your official ballot & tracker</Loading>
+            <Loading>Printing your official ballot</Loading>
```

Comment:
I'd like to keep the indication that the tracker is being printed. I can see this being split into a second screen at some later point, but for now keeping the indication in this text seems best to me.